File: QuanLyCuaHangUXUI/QuanLyCuaHangProject/src/models/dto/dataCharts/DetailProductDC.js
class DetailProductDC {
  constructor(
    maCtsp,
    maSp,
    kichThuoc = null,
    huongVi = null,
    soLuongTon = null,
    donGia = null,
    tenHinhAnh = null,
  ) {
    this.maCtsp = maCtsp
    this.maSp = maSp
    this.kichThuoc = kichThuoc
    this.huongVi = huongVi
    this.soLuongTon = soLuongTon
    this.donGia = donGia
    this.tenHinhAnh = tenHinhAnh
  }
}

export default DetailProductDC

File: APIQuanLyCuaHang/APIQuanLyCuaHang/DTO/DataChart/DetailProductDC.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace APIQuanLyCuaHang.DTO.DataChart
{
    public class DetailProductDC
    {
        public int MaCtsp { get; set; }

        public int MaSp { get; set; }

        public string? KichThuoc { get; set; }

        public string? HuongVi { get; set; }

        public int? SoLuongTon { get; set; }

        public decimal? DonGia { get; set; }
        public string? TenHinhAnh { get; set; }
    }
}